bin/mage.bb

```diff
@@ -75,10 +75,18 @@
           (try (bt/run task)
                (catch Exception e
                  (let [message (ex-message e)
-                       data (ex-data e)]
-                   (when message (println (c/red    (c/reverse-color "Error     :")) message))
-                   (when data    (println (c/yellow (c/reverse-color "Error data:")) data))
-                   (when e       (println (c/yellow "\nException:\n") e))
```

Comment:
when there is a message and / or data, there will always be an exception: so stop printing it twice